latex/expressingParallelism.tex

```diff
@@ -1868,8 +1868,10 @@ \subsection{Kernel class}
 {info::kernel_work_group::preferred_work_group_size_multiple}
 {size_t}
 {
-  Returns the preferred work-group size for executing a kernel on a particular
-  device.
+  Returns a value, of which work-group size is preferred to be a multiple,
+  for executing a kernel on a particular device.  This is a performance
+  hint.  The value must be less than or equal to that returned by
+  \codeinline{info::kernel_work_group::work_group_size}.
```

Comment:
Unrelated to your change, but seems to me 

> info::kernel_work_group::work_group_size

should be named 

>info::kernel_work_group::max_work_group_size 

instead?

Reply:
I thought so too, but SYCL currently matches OpenCL.  The device has CL_DEVICE_MAX_WORK_GROUP_SIZE, while kernels have CL_KERNEL_WORK_GROUP_SIZE which is the maximum.